docs/core/integration-quality-scale/rules/iqs023.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "IQS023 - Set Parallel updates"
+---
+
+IQS023 is as follows:
+
+Set Parallel updates
+
+## Reasoning
+
+Some devices or services don't like receiving a lot of requests at the same time.
+So Home Assistant has a built-in feature to limit the number of requests that are sent to a device or service at the same time.
```

Comment:
```suggestion
To avoid that, Home Assistant has a built-in feature to limit the number of requests that are sent to a device or service at the same time.
```